services/app-api/libs/kafka-source-lib.js

```diff
@@ -40,7 +40,7 @@ class KafkaSourceLib {
   */
 
   unmarshallOptions = {
-    convertEmptyValues: true,
+    convertWithoutMapWrapper: false,
```

Comment:
this should be false by default but I'm just making it explicit what our options are